lib/services.js

```diff
@@ -2,7 +2,12 @@ var fs = require('fs');
 var ejs = require('ejs');
 
 ejs.filters.q = function(obj) {
-  return JSON.stringify(obj, null, 2 );
+  // to handle strings being returned with double-quotes
+  // but linter requires single quotes for strings
+  if (typeof obj == 'string' && !/\\'/.test(obj))
+    return '\'' + obj + '\'';
+  else
+    return JSON.stringify(obj, null, 2);
```

Comment:
In that case we will need the other PR to land before this, it will cause hundreds of Lint errors on the generated code. 

@superkhau right now I have two somewhat closely related tasks in this PR
- linting generated code
- changing the linter to eslint
so do you think I should separate them into PR as such? 
please advise